sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -9,19 +9,15 @@
       Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.
       Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Compute" (https://www.nuget.org/packages/Azure.ResourceManager.Compute/). Please see the package changelog for more information.
     </Description>
-    <Version>47.0.1</Version>
+    <Version>48.0.0</Version>
     <AssemblyName>Microsoft.Azure.Management.Compute</AssemblyName>
     <PackageTags>management;virtual machine;compute;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
         This is a public release of the Azure Compute SDK. Included with this release is:
-        1. Restore Points API (Restore Points API model uses properties bag).
```

Comment:
Please include: 
PublicAddressSKu object values were changed from 'publicIPAddressSkuName' and 'publicIPAddressSkuTier' to 'name' and 'tier' respectively.

SecurityTypes enum was changed to be string based.

 Updated StorageAccountType object description and added to the ManagedDiskParameters property StorageAccountType description.


Also please keep this one:
1. Restore Points API (Restore Points API model uses properties bag).

This one was updated and the updated version was not included in the last release.